src/admin/content-block/components/ContentBlockFieldEditor/ContentBlockFieldEditor.tsx

```diff
@@ -0,0 +1,76 @@
+import { get } from 'lodash-es';
+import React, { FunctionComponent } from 'react';
+
+import { SelectOption } from '@viaa/avo2-components';
+
+import { EDITOR_TYPES_MAP } from '../../content-block.const';
+import {
+	ContentBlockComponentState,
+	ContentBlockEditor,
+	ContentBlockField,
+	ContentBlockMeta,
+	ContentBlockState,
+	ContentBlockStateType,
+} from '../../content-block.types';
+
+interface ContentBlockFieldProps {
+	block: ContentBlockMeta;
+	fieldKey: keyof ContentBlockComponentState | keyof ContentBlockState;
+	field: ContentBlockField;
+	type: ContentBlockStateType;
+	state: ContentBlockComponentState | ContentBlockState;
+	stateIndex?: number;
+	handleChange: (
+		formGroupType: ContentBlockStateType,
+		fieldKey: keyof ContentBlockComponentState | keyof ContentBlockState,
+		value: any,
+		stateIndex?: number
+	) => void;
+}
```

Comment:
mss moeten we bij deze props nog wat commentaar zetten met een voorbeeld van de waarde, zodat we makkelijker weten waarvoor iedere prop precies dient